.gitignore

```
@@ -32,3 +32,5 @@
 .DS_Store
 .idea/*
 .vscode/*
+
+stdlib/*/Manifest.toml
```

```suggestion
/stdlib/*/Manifest.toml
```
But would probably be more canonical to put `/*/Manifest.toml` in the existing `stdlib/.gitignore`?

---

yup indeed. Done in latest